examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -116,3 +117,16 @@ void emberAfLowPowerClusterInitCallback(EndpointId endpoint)
     ChipLogProgress(Zcl, "TV Linux App: LowPower::SetDefaultDelegate");
     chip::app::Clusters::LowPower::SetDefaultDelegate(endpoint, &lowPowerManager);
 }
+
+void MatterPostAttributeChangeCallback(const chip::app::ConcreteAttributePath & attributePath, uint8_t mask, uint8_t type,
+                                       uint16_t size, uint8_t * value)
+{
+    switch (attributePath.mClusterId)
+    {
+    case chip::app::Clusters::WindowCovering::Id:
+        MatterWindowCoveringClusterServerAttributeChangedCallback(attributePath);
```

Comment:
I assume this behavior is not Linux-specific, right?

The way this should work, and I know it's non-obvious, is that in `src/app/zap-templates/templates/app/helper.js` the cluster name gets added to the `endpointClusterWithAttributeChanged` array.  That will codegen the right bits that `MatterWindowCoveringClusterServerAttributeChangedCallback` will be automatically called by the SDK, instead of the application having to do it.  Then the application's `MatterPostAttributeChangeCallback` is just used for the state the application wants to maintain.

Reply:
I was wondering that indeed this is not cluster specific and this why i have added a weak reference you pointed later
Ok gonna have look for an update thk

Reply:
Removed following your advice